packages/@aws-cdk/aws-apigateway/lib/method.ts

```diff
@@ -73,6 +73,7 @@ export interface MethodOptions {
 
   /**
    * The ID of the associated request validator.
+   * @deprecated Use `reqValidator` to pass validator options
```

Comment:
Actually, let's not deprecate this yet. Let's keep this around. It'll support cases when a `RequestValidator` is imported, or a user wants to use the same request validator in different methods.

I'm not sure yet that these are actual use cases, that people want.